content/posts/matplotlib/custom-3d-engine/index.md

```diff
@@ -88,15 +88,15 @@ plt.show()
 
 You should obtain something like this ([bunny-1.py](bunny-1.py)):
 
-![](bunny-1.png)
+![ ](bunny-1.png)
 
 # Perspective Projection
 
 The rendering we've just made is actually an [orthographic
 projection](https://en.wikipedia.org/wiki/Orthographic_projection) while the
 top bunny uses a [perspective projection](https://en.wikipedia.org/wiki/3D_projection#Perspective_projection):
 
-![](projections.png)
+![ ](projections.png)
```

Comment:
```suggestion
![Difference in perspective projection and orthographic projection. The near clip plane appears smaller in the perspective projective than in the orthographic projection.](projections.png)
```

Reply:
This is a tough image indeed. I'm not confident how I'd describe it even after reviewing the blog post, so I'm sure other people have other opinions.

I think you did a good job recognizing that while there's a lot of information in the image, not a lot of the information is referenced in the blog. I don't think I have anything I'd change because the blog links out to more information on perspective and orthographic projection, so the author seems to agree that understanding it in depth is not the most important part.

In general, I've heard different ways to measure "too much" description. Personally, I am more concerned with making sure the amount of information matches than counting characters. I roughly cut alt text at three sentences long, and if it needs to be longer because it holds more content we find a way to link to a longer description.